digitalocean-app-platform/README.md

```diff
@@ -0,0 +1,74 @@
+a focus on the containers and their setup in a given environment, and more of a focus on deploying the application for immediate access. After testing
+the deployment of the SCIM bridge with App Platform, it is now being added as a deployment option for those using DigitalOcean to host the SCIM bridge.
+
+Deploying the SCIM bridge with App Platform comes with a few benefits:
+* App Platform will provide and host the URL for your SCIM bridge; you will not need to setup an A record or prepare a name for a URL as noted in [PREPARATION.md](https://github.com/1Password/scim-examples/blob/master/PREPARATION.md)
+* App Platfom will host the SCIM application for a low cost of $5/mo. An additional $5/mo or $6/mo will be utilized for the Droplet created for the Redis container.
+* There's no need to manage the container that the SCIM bridge will be running on.
+
+
+## Preparation and Deployment
+To get started with deploying the SCIM bridge using App Platform, you'll need:
+
+* Access to your organization's DigitalOcean tenant.
+* Access to your organization's Github account in order to fork this repository.
+* Access to create a Droplet for Redis in your organization's DigitalOcean tenant.
+
+
+### Step One: Setting up your Redis Droplet
+
+Before you deploy the SCIM bridge application using App Platform, a redis database must be created first, so that you can add the connection details for your database to the application at setup.
+
+To create a Droplet:
+
+* Under Manage in the left-hand navigation menu, select Droplet or select the Create dropdown menu in the top right corner of your DigitalOcean tenant and select Droplet.
+* Choose an image for your container.
+* Choose a plan for your Droplet. (The Basic (shared CPU) tier is sufficient, but choose what's best for your organization)
+* Choose a datacenter region.
+* Under Finalize and Create, you will only need 1 Droplet.
+* Once you've configured the other settings on this page to your liking, click Create Droplet.
+
+Once the creation process of your Droplet is complete:
+
+* Click on the hostname of your new container from your list of Droplets.
+* Click on Console. (ensure that the credentials for your image are set and that you can log into the container)
+* At this point, you will want to install redis on your Droplet. DigitalOcean provides detailed documentaion on how to install redis on each of its provided images. Documentation can be found [here](https://www.digitalocean.com/community/tutorial_collections/how-to-install-and-secure-redis)
+* For the ```Binding to Localhost``` step in the redis documentation, you will want to ensure that you allow all connections initially, so that the SCIM application can make a connection to your Droplet. After the successful deployment of your SCIM application, you can lock down access to your redis Droplet, ensuring that your SCIM application only has access to that Droplet.
+
+### Step Two: Building and Deploying using App Platform
```

Comment:
Did you determine that their [hosted Redis](https://www.digitalocean.com/products/managed-databases-redis/) is not a viable solution for this?

Reply:
done.